lib/puma/server.rb

```diff
@@ -28,6 +28,9 @@ class Server
     include Puma::Const
     extend  Puma::Delegation
 
+    # TCP_TIME_WAIT: 6, TCP_CLOSE: 7, TCP_CLOSE_WAIT: 8, TCP_LAST_ACK: 9
+    IGNORED_CHECK_CLIENT_SOCKET_STATES = (6..9)
```

Comment:
The enum for TCP states is defined [here](https://github.com/torvalds/linux/blob/master/include/net/tcp_states.h#L16).